Firmware/Marlin_main.cpp

```diff
@@ -522,7 +522,8 @@ bool __attribute__((noinline)) printer_active() {
         || (lcd_commands_type != LcdCommands::Idle)
         || MMU2::mmu2.MMU_PRINT_SAVED()
         || homing_flag
-        || mesh_bed_leveling_flag;
+        || mesh_bed_leveling_flag
+        || (eeprom_read_byte((uint8_t*)EEPROM_UVLO) != PowerPanic::NO_PENDING_RECOVERY);
```

Comment:
Reading the eeprom takes some time, so any call to `printer_active` will get slower. Have you considered that?

Reply:
in such a case it is probably ok and no changes are really necessary :+1: